source/_integrations/lg_thinq.markdown

```diff
@@ -98,6 +100,67 @@ A read-only property which has only two states that can be toggled is represente
 | | Enable Sabbath mode |
 | [Wine Cellar](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Wine-Cellar/) | Enable Sabbath mode |
 
+#### Sensor
+
```

Comment:
Is this list complete? Asking because in the code, I see sensors such as brewing duration or brewing period, which I don't see in this list.

Reply:
Modified